src/server/kiwix-serve.cpp

```diff
@@ -39,6 +39,7 @@
 #endif
 
 #include "../version.h"
+using namespace std;
```

Comment:
Please don't. Add explicit `std::` when it is needed (`std::vector`, `std::string`, ....) instead.